api/definition/indicators/paths.yaml

```
@@ -80,14 +81,26 @@ indicators_by_indicator_id_data:
       - indicators
     summary: add a batch of new data for an indicator
     description: Not Yet Implemented - Creates new data for the indicator. The data is always created in an unpublished and unapproved state.
+    requestBody:
+      required: true
+      content:
+        multipart/form-data:
+          schema:
+            type: object
+            properties:
+              file: 
+                type: string
+                format: binary
     responses:
-      "501":
-        description: Not Implemented
+      "200":
```

Should we use `202` instead? I think it's well suited to this purpose.
```suggestion
      "202":
```

---

I did think we'd swap to that later, but happy to make the change now.